GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1534 @@
+
+            #endregion
+        }
+
+        private static async Task<User> CreateNewUser(IActiveDirectoryClient client, VerifiedDomain defaultDomain)
+        {
+            #region Create a new User
+
+            // **********************************************************
+            // Requires Directory.ReadWrite.All or Directory.AccessAsUser.All, and the signed in user
+            // must be a privileged user (like a company or user admin)
+            // **********************************************************
+
+            User newUser = new User();
+            if (defaultDomain.Name != null)
+            {
+                Console.WriteLine("\nCreating a new user...");
+                Console.WriteLine("\n  Please enter first name for new user:");
+                String firstName = Console.ReadLine();
+                Console.WriteLine("\n  Please enter last name for new user:");
+                String lastName = Console.ReadLine();
+                newUser.DisplayName = firstName + " " + lastName;
+                newUser.UserPrincipalName = firstName + "." + lastName + Helper.GetRandomString(4) + "@" +
+                                            defaultDomain.Name;
+                newUser.AccountEnabled = true;
+                newUser.MailNickname = firstName + lastName;
+                newUser.PasswordProfile = new PasswordProfile
+                {
+                    Password = "ChangeMe123!",
+                    ForceChangePasswordNextLogin = true
+                };
+                newUser.UsageLocation = "US";
+                try
+                {
+                    await client.Users.AddUserAsync(newUser);
+                    Console.WriteLine("\nNew User {0} was created", newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError creating new user {0} {1}", e.Message,
+                        e.InnerException != null ? e.InnerException.Message : "");
+                }
+            }
+            return newUser;
+
+            #endregion
+        }
+
+        private static async Task UpdateNewUser(IUser newUser)
+        {
+            #region Update User properties
+
+            //*******************************************************************************************
+            // update the newly created user's Password, PasswordPolicies and City
+            //*********************************************************************************************
+            if (newUser.ObjectId != null)
+            {
+                // update User's info
+                newUser.City = "Seattle";
+                newUser.Country = "UK";
+                newUser.Mobile = "+4477889456789";
+                newUser.UserType = "Member";
+
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.WriteLine("\nUser {0} was updated", newUser.DisplayName);
```

Comment:
ok.

---
In reply to: [80543147](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r80543147) [](ancestors = 80543147)